python/nammu/controller/NammuController.py

```diff
@@ -432,27 +458,37 @@ def validate(self, event=None):
         self.atfAreaController.clearToolTips()
 
         if self.currentFilename:
-            self.logger.debug("Validating ATF file %s.", self.currentFilename)
+            # Get the file extension of the current file
+            file_ext = os.path.splitext(self.currentFilename)[1]
 
-            # Search for project name in file. If not found, don't validate
-            project = self.get_project()
+            if file_ext == '.atf':
+                self.logger.debug("Validating ATF file %s.",
```

Comment:
Now that you're changing this, it would be nice to make logger notation consistent with code elsewhere and replace `%s` with `{}`.

Reply:
I think this change should be its own branch. I have opened issue #281 to address this and will make these changes in a new PR after #277 and #280 are closed.